OpenOversight/app/utils.py

```diff
@@ -249,15 +249,8 @@ def filter_by_form(form, officer_query, is_browse_filter=False):
                                                 Officer.birth_year == None))  # noqa
 
     officer_query = officer_query.outerjoin(Assignment)
-    if form['rank'] == 'PO':
-        officer_query = officer_query.filter(
-            db.or_(Assignment.rank.like('%%PO%%'),
-                   Assignment.rank.like('%%POLICE OFFICER%%'),
-                   Assignment.rank == 'Not Sure')  # noqa
```

Comment:
I can write this migration (unless someone else wants to) and append in a commit when this is ready for merge (just so that we don't merge something into `develop` that we can't yet deploy), it should update ranks named exactly "PO" to "POLICE OFFICER"